openlobby/settings.py

```diff
@@ -8,7 +8,7 @@
 DEBUG = 'DEBUG' in os.environ
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY')
+SECRET_KEY = os.environ.get('SECRET_KEY','very-secret-key')
```

Comment:
Secret key should not have default if server is not running in debug mode. It must be set from outside environment in production. Below is check for forgotten secret key, which will not work with default value.